src/plugins/vis_augmenter/public/utils/utils.ts

```diff
@@ -13,10 +13,25 @@ import {
 } from '../../../../plugins/expressions/public';
 import { ISavedAugmentVis, SavedAugmentVisLoader, VisLayerFunctionDefinition } from '../';
 
-// TODO: provide a deeper eligibility check.
-// Tracked in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3268
-export const isEligibleForVisLayers = (vis: Vis): boolean => {
-  return vis.params.type === 'line';
+export const isEligibleForVisLayers = (vis: Vis, dimensions: VislibDimensions): boolean => {
+  // Only support date histogram and ensure there is only 1 x-axis and it has to be on the bottom
+  const isValidXaxis =
+    vis.data.aggs?.byTypeName('date_histogram').length === 1 &&
+    vis.params.categoryAxes.length === 1 &&
+    vis.params.categoryAxes[0].position === 'bottom';
```

Comment:
This is because we want to force it to be a bottom axis for the use case of the functionality. This would make it so it is aligned with the augmented triangles that indicate the alerts/anomalies.

Reply:
got it, makes sense. In the future, should be easy enough to allow it on top, too (put both the axis and the triangles up there), if that's something important to users.

Reply:
We need to have it on the bottom since the augmentation from the alerts/anomalies would look odd when the axis is not on the bottom.